.stylelintrc.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
- [ ] Let's add a descriptive Readme file in your project that explains what you did in the project along with author name and technologies used. Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

Reply:
Thank you for the kind reminder, I'll add it